.ci/maven-prepare.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -z $1 ]]; then
+  echo "latest version is not set"
+  exit 1
+fi
+LATEST_VERSION=$1
+echo LATEST_VERSION="$LATEST_VERSION"
+
+POM_FILE=pom.xml
+CURRENT_VERSION=$(grep "^  <version>.*</version>$" $POM_FILE | awk -F'[><]' '{print $3}')
```

Comment:
Use xmlstartlet to certain on version from pom.xml.
In pom version is with '-SNAPSHOT'.

Reply:
Changes Done

Reply:
Changes Done